pallets/subtensor/src/coinbase/run_coinbase.rs

```diff
@@ -78,7 +78,7 @@ impl<T: Config> Pallet<T> {
                 );
                 if let Ok(buy_swap_result_ok) = buy_swap_result {
                     let bought_alpha: AlphaCurrency = buy_swap_result_ok.amount_paid_out.into();
-                    Self::recycle_subnet_alpha(*netuid_i, bought_alpha);
+                    Self::distribute_alpha_to_subnet_owners(*netuid_i, bought_alpha);
```

Comment:
I suggest a global hyperparameter to allow root to choose between `0` for full recycle and `65535` for full distribution. We don't know yet how powerful this is going to be.